.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@ What's changed, or what was fixed?
 **Target Live Date:** YYYY-MM-DD
 
 - [ ] This has been reviewed and approved by (NAME)
-- [ ] I have run `gulp test` locally and all tests pass.
+- [ ] I have run `npm test` locally and all tests pass.
```

Comment:
Drive-by improvement: `npm test` still works even if `gulp` is not installed globally.

Reply:
👍